docs/source/migration.mdx

```diff
@@ -280,7 +282,18 @@ The Apollo Server core team no longer maintains the following integration packag
 * [`apollo-server-cloudflare`](https://www.npmjs.com/package/apollo-server-cloudflare)
 * [`apollo-server-azure-functions`](https://www.npmjs.com/package/apollo-server-azure-functions)
 
-If you use one of these packages, see [the community integration index](./integrations/integration-index/) to see if an integration exists for your environment yet, or [build your own](./integrations/building-integrations).
+Instead, we now work with the broader community to maintain the following open-source integrations for Apollo Server:
+
+| Web Framework | Integration Package |
+|--------|-------------|
+| [AWS Lambda](https://aws.amazon.com/lambda/) | [`@as-integrations/aws-lambda`](https://www.npmjs.com/package/@as-integrations/aws-lambda) |
+| [Fastify](https://fastify.io/) | [`@as-integrations/fastify`](https://www.npmjs.com/package/@as-integrations/fastify) |
+| [Hapi](https://hapi.dev/) | [`@as-integrations/hapi`](https://www.npmjs.com/package/@as-integrations/hapi) |
+| [Koa](https://koajs.com/) | [`@as-integrations/koa`](https://www.npmjs.com/package/@as-integrations/koa) |
+| [Next.js](https://nextjs.org) | [`@as-integrations/next`](https://www.npmjs.com/package/@as-integrations/next) |
+| [Nuxt](https://v3.nuxtjs.org/) / [h3](https://github.com/unjs/h3) | [`@as-integrations/h3`](https://www.npmjs.com/package/@as-integrations/h3) |
+
+> See [the community integration index](./integrations/integration-index/) for the most up-to-date list of current integrations. If an integration doesn't exist for your framework, you can always [build your own](./integrations/building-integrations)!
```

Comment:
and if you do that you probably don't even need the link to integration-index here.

Reply:
Removed this link too :)